tasks/base64-encode-checksum/base64-encode-checksum.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: base64-encode-checksum
+  labels:
+    app.kubernetes.io/version: "0.2.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Task to encode the Terraform Checksum file
+  params:
+    - name: binaries_dir
+      type: string
+      description: Path where the binaries are stored in the workspace
+      default: "binaries"
+    - name: subdirectory
+      description: Subdirectory inside the workspace to be used
+      type: string
+      default: ""
+  workspaces:
+    - name: data
+      description: The workspace where the snapshot is stored. The extracted binaries will be stored here as well.
+  results:
+    - name: blob
+      type: string
+      description: Base64 encoded file
```

Comment:
If it is a file, shouldn't it be in a workspace so downstream things can use it?

Reply:
I changed the description a bit. It's not a file, but the output of the file after being base64 encoded